geth/common/types.go

```diff
@@ -163,11 +163,14 @@ type QueuedTx struct {
 type SendTxArgs struct {
 	From     common.Address  `json:"from"`
 	To       *common.Address `json:"to"`
-	Gas      *hexutil.Big    `json:"gas"`
+	Gas      *hexutil.Uint64 `json:"gas"`
 	GasPrice *hexutil.Big    `json:"gasPrice"`
 	Value    *hexutil.Big    `json:"value"`
-	Data     hexutil.Bytes   `json:"data"`
 	Nonce    *hexutil.Uint64 `json:"nonce"`
+	// We accept "data" and "input" for backwards-compatibility reasons. "input" is the
+	// newer name and should be preferred by clients.
+	Data  *hexutil.Bytes `json:"data"`
```

Comment:
Do we need to change the data type to `*hexutil.Bytes`?

Reply:
I don't think so, I was trying to keep it similar to geth's struct. Will change it to `Input hexutil.Bytes`.